Description: Conditionally skip copying jquery.
 This patch tried to reduce the footprint of the generated docs and to avoid
 embedding unnecessary JavaScript libraries but it breaks things.
 It should be disabled or deleted.
Forwarded: https://github.com/doxygen/doxygen/issues/7367
Author: Paolo Greppi <paolo.greppi@libpf.com>
Last-Update: 2019-10-31
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/

Index: doxygen/src/htmlgen.cpp
===================================================================
--- doxygen.orig/src/htmlgen.cpp
+++ doxygen/src/htmlgen.cpp
@@ -775,7 +775,10 @@ void HtmlGenerator::init()
   {
     mgr.copyResourceAs("fixed_tabs.css",dname,"tabs.css");
   }
-  mgr.copyResource("jquery.js",dname);
+  if (Config_getBool(GENERATE_TREEVIEW) || Config_getBool(SEARCHENGINE))
+  {
+    mgr.copyResource("jquery.js",dname);
+  }
   if (Config_getBool(INTERACTIVE_SVG))
   {
     mgr.copyResource("svgpan.js",dname);
